content/docs/2.13/authentication-providers/aws.md

```diff
@@ -0,0 +1,79 @@
++++
+title = "Pod Identity Webhook for AWS by using IAM Roles for Service Accounts (IRSA)"
++++
+
+[**AWS IAM Roles for Service Accounts (IRSA) Pod Identity Webhook**](https://github.com/aws/amazon-eks-pod-identity-webhook) ([details](https://aws.amazon.com/blogs/opensource/introducing-fine-grained-iam-roles-service-accounts/)) allows you to provide the role name using an annotation on a service account associated with your pod.
+
+You can tell KEDA to use AWS Pod Identity Webhook via `podIdentity.provider`.
+
+```yaml
+podIdentity:
+  provider: aws # Optional. Default: none
+  roleArn: <role-arn|workload> # Optional. Default: RoleArn from annotation on service-account.
```

Comment:
>Default: RoleArn from annotation on service-account.

This would imply that the default is `workload` right?

Reply:
No no, default is KEDA's role

Reply:
Then it's probably more clear to say something like

```suggestion
  roleArn: <role-arn|workload> # Optional. If not set the IAM role which the KEDA operator uses will be used
```